packages/android-foreground-service/src/definitions.ts

```diff
@@ -218,3 +241,80 @@ export interface PermissionStatus {
  * @since 6.1.0
  */
 export type UpdateForegroundServiceOptions = StartForegroundServiceOptions;
+
+/**
+ * @since 6.1.0
+ */
+export interface Channel {
```

Comment:
```suggestion
export interface CreateNotificationChannelOptions {
```

Let's rename the `Channel` interface to `CreateNotificationChannelOptions`, since `Channel` is not used anywhere else. We can also introduce a separate interface later without breaking change.